optimize/pom.xml

```diff
@@ -14,8 +14,8 @@
     <camunda.maven.artifacts.version>7.21.0</camunda.maven.artifacts.version>
 
     <!-- We use this version to compile but run IT against containers with zeebe.docker.version -->
-    <zeebe.version>8.6.0-alpha5</zeebe.version>
-    <identity.version>8.6.0-alpha5</identity.version>
+    <zeebe.version>8.6.8</zeebe.version>
+    <identity.version>8.6.8</identity.version>
```

Comment:
🤔 Shouldn't this just use something like `${project.version}`, i.e. `8.6.1-SNAPSHOT`?

Reply:
On 8.6 and 8.7, Optimize is not part of the single application, and doesn't inherit from the project root pom. I think the project version property here would therefore refer to the Optimize version, which are versioned independently from Identity and Zeebe

Reply:
> Also, why is it 8.6.1-SNAPSHOT on stable/8.7

I guess it could also be `8.7.0-SNAPSHOT` in this case. Either way, I would say it doesn't really matter, given that the Optimize 8.7 version is maintained from the `stable/optimize-8.7` branch rather than here